omnibus/files/private-chef-ctl-commands/rotate_credentials.rb

```diff
@@ -122,6 +137,77 @@
   exit(0)
 end
 
+add_command_under_category "require-credential-rotation", "credential-rotation", "Disable the Chef Server and require credential rotation", 2 do
+  @agree_to_disable = false
+  @ui = HighLine.new
+
+  OptionParser.new do |opts|
+    opts.banner = "Usage: chef-server-ctl require-credential-rotation [--yes]"
+
+    opts.on("-h", "--help", "Show this message") do
+      puts opts
+      exit
+    end
+
+    opts.on("-y", "--yes", "Agree to disable the Chef Server and require credential rotation") do
+      @agree_to_disable = true
+    end
+  end.parse!(ARGV)
+
+  # Agree to disable and require rotation
+  msg = "Are you sure you want to disable the Chef Server and require credential rotation?" \
+          "\n Type 'yes' to confirm: "
+  exit(1) unless @agree_to_disable || @ui.ask("<%= color(%Q(#{msg}), :yellow) %>") =~ /yes/i
+
+  # Shut down Chef Server
+  run_sv_command("stop")
```

Comment:
You _cant_ restart the server until the secrets have been rotated which is very much intentional.  When you do rotate the secrets the Chef Server will come online again.